modules/_node-scripts/util/constants.mjs

```diff
@@ -65,11 +69,9 @@ export async function getProjectDirs(dir = undefined) {
 		) {
 			projectDirs.push(path.resolve(dir));
 			break;
-		}
-		else if (NO_RECURSE_PROJECT_DIRS.includes(dirent.name)) {
+		} else if (NO_RECURSE_PROJECT_DIRS.includes(dirent.name)) {
```

Comment:
This should be SF'd

Reply:
new line if/else

Reply:
Yeah, the PR is failing SF. I need to fix it once https://github.com/liferay-frontend/liferay-portal/pull/4248 is merged.